neo/Consensus/ConsensusContext.cs

```diff
@@ -29,20 +29,19 @@ internal class ConsensusContext : IConsensusContext
         public UInt160 NextConsensus { get; set; }
         public UInt256[] TransactionHashes { get; set; }
         public Dictionary<UInt256, Transaction> Transactions { get; set; }
-        public UInt256[] Preparations { get; set; }
-        public byte[][] Commits { get; set; }
-        public byte[] ExpectedView { get; set; }
-        private Snapshot snapshot;
+        public ConsensusPayload[] PreparationPayloads { get; set; }
+        public ConsensusPayload[] CommitPayloads { get; set; }
+        public ConsensusPayload[] ChangeViewPayloads { get; set; }
```

Comment:
Should we make the order of these members match the `IConsensusContext`, and move the ChangeViewPayloads up above the members related to the `PrepareRequest`?

Reply:
I changed the order of `IConsensusContext`.

Reply:
Great thanks.